src/index.html

```diff
@@ -14,105 +14,151 @@
     <header class="header">
       <h1>riddle me this…</h1>
     </header>
+    <main class="">
+      <section class="page app-grid">
+        <section class="card" data-js="card">
+          <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
+          <h2>
+            How do you make the number 7 even without addition, subtraction,
+            multiplication, or division?
+          </h2>
+          <details>
+            <summary class="card__show-answer">Show Answer</summary>
```

Comment:
I would prefer to be a button as this is semantically not really a summary of the text below.